setup_miniconda.sh

```diff
@@ -40,27 +48,84 @@ then
 
   # Create an Environment (EXAMPLE: creating an environment called
   # pyfuncebletesting with Python version "pythonVersion")
-  conda create -y -q -n pyfuncebletesting python="${pythonVersion}"
+  conda create -y -q -n "${pyfunceblePackageName}" python="${pythonVersion}"
 
   # Activate this environment you just created
   # According to the https://docs.conda.io/projects/conda/en/latest/_downloads/843d9e0198f2a193a3484886fa28163c/conda-cheatsheet.pdf
   # We shall replace source with conda activate vs source
-  conda activate pyfuncebletesting
+  conda activate "${pyfunceblePackageName}"
 
   # Query Python and Pip versions inside this environment
   python -VVV
   pip --version
 
   # Install PyFunceble in this environment (pyfuncebletesting)
   pip -q install --no-cache-dir -U "${pyfunceblePackageName}"
+  pip -q install --no-cache-dir git+https://github.com/Ultimate-Hosts-Blacklist/whitelist.git@script
+
+  # Test Pyfunceble version installed
+  pyfunceble --version
+  uhb-whitelist --version
 
   # prepared for installing thru conda package
   #conda install -c pyfunceble "${pyfunceblePackageName}"
 
+  # Copy default .pyfunceble-env to new enviroment
+  cp "$HOME/.config/PyFunceble/.pyfunceble-env" \
+    "${condaInstallDir}/envs/${pyfunceblePackageName}"
+
   # When finished - Deactivate the environment
   conda deactivate
+
 else
-  echo "We assume that conda is already installed."
+  echo "We assume that Pyfunceble envs is already installed."
+fi
+
+if [[ ! -d ${condaInstallDir}/envs/pyfunceble-dev ]]
+then
+  # Setup pyfunceble@ version to be used
+  pyfunceblePackageName="pyfunceble-dev"
+
+  # Get the conda CLI.
+  source "${condaInstallDir}/etc/profile.d/conda.sh"
+
+  hash conda
+
+  # Update Conda
+  conda update -q conda
+
+  # Create an Environment (EXAMPLE: creating an environment called
+  conda create -y -q -n "${pyfunceblePackageName}" python="${pythonVersion}"
+
+  # Activate this environment you just created
+  # According to the https://docs.conda.io/projects/conda/en/latest/_downloads/843d9e0198f2a193a3484886fa28163c/conda-cheatsheet.pdf
+  # We shall replace source with conda activate vs source
+  conda activate "${pyfunceblePackageName}"
+
+  # Query Python and Pip versions inside this environment
+  python -VVV
+  pip --version
+
+  # Install PyFunceble in this environment (pyfuncebletesting)
+  pip -q install --no-cache-dir -U "${pyfunceblePackageName}"
+  pip -q install --no-cache-dir git+https://github.com/Ultimate-Hosts-Blacklist/whitelist.git@script-dev
+
+  # prepared for installing thru conda package
+  #conda install -c pyfunceble "${pyfunceblePackageName}"
```

Comment:
Hey @funilrys 

According to this <https://github.com/funilrys/PyFunceble/commit/7ff6e5e2df8663af969134809f363f8a24a9b557> should this then be changed into `conda install -c conda-forge -c pyfunceble pyfunceble=3` ?